cmd/server/http/middleware/prometheus.go

```diff
@@ -1,18 +0,0 @@
-package middleware
-
-import (
-	"github.com/gorilla/mux"
-	"github.com/slok/go-http-metrics/metrics/prometheus"
-	"github.com/slok/go-http-metrics/middleware"
-	"net/http"
-)
-
-func Prometheus() mux.MiddlewareFunc {
```

Comment:
why getting rid of this ?

Reply:
in general, we delete code when we move it in another location or because we don't need anymore

Reply:
then explain where it has been moved or explain that it is not used anymore 👍

Reply:
nop, im not going to explain all code modification